### DISCUSSION TITLE
Analyze without Protease (peptidomic)

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#369] [Q&A]
Hi Vadim,

Is it possible to perform a DIA-NN analyse from a fasta file without use any Protease?

Thank in advance,

Benoit

**Comment by vdemichev** [comment]
Hi Benoit,

Not a proper set of protein sequences. What's possible is to supply DIA-NN with a list of peptides (formatted as FASTA) and turn off the protease with " -- cut " (without quotes) added to the "Additional options" textbox.

Best,
Vadim

**Reply by BenoitBe** [reply]
Hi Vadim,

I am trying this kind of research with a list of peptide formated as fasta but apparently the " -- cut " (without quotes) added to the "Additional options" textbox don't work:

WARNING: unrecognised option [--cut]

Do I have misswriting the modif or maybe not work with recent version of soft?

Thax a lot in advance,

Benoit

**Reply by vdemichev** [reply]
Hi Benoit,

Can you please share the whole top part of the log?

Best,
Vadim

**Reply by vdemichev** [reply]
I just noticed that I wrote incorrectly, apologies, should ne --cut, not -- cut, i.e. no space after double dash.

**Reply by BenoitBe** [reply]
Yep, work better, thanks a lot!!
